src/main/java/io/reactivex/Flowable.java

```diff
@@ -194,7 +194,7 @@ public static int bufferSize() {
      */
     @SchedulerSupport(SchedulerSupport.NONE)
     @BackpressureSupport(BackpressureKind.FULL)
-    public static <T, R> Flowable<R> combineLatest(Function<? super T[], ? extends R> combiner, Publisher<? extends T>... sources) {
+    public static <T, R> Flowable<R> combineLatest(Function<? super Object[], ? extends R> combiner, Publisher<? extends T>... sources) {
```

Comment:
This and other overloads of operators let's you use varargs to specify any number of sources at the expense that varargs has to be the last argument.